recipes/tabulate/all/conanfile.py

```diff
@@ -1,57 +1,69 @@
-import os
 from conans import ConanFile, tools
 from conans.errors import ConanInvalidConfiguration
-from conans.tools import Version
+import os
+
+required_conan_version = ">=1.43.0"
 
-class Tabulate(ConanFile):
+
+class TabulateConan(ConanFile):
     name = "tabulate"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/p-ranav/tabulate"
     description = "Table Maker for Modern C++"
-    settings = "compiler"
     topics = ("header-only", "cpp17", "tabulate", "table", "cli")
     license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = self.name + "-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
-
-    def configure(self):
-        compiler = str(self.settings.compiler)
-        compiler_version = tools.Version(self.settings.compiler.version)
-
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, "17")
-        else:
-            self.output.warn("%s recipe lacks information about the %s compiler"
-                             " standard version support" % (self.name, compiler))
+    @property
+    def _min_cppstd(self):
+        return "17" if tools.Version(self.version) < "1.3.0" else "11"
 
-        minimal_version = {
+    @property
+    def _min_compiler_cpp17(self):
+        return {
             "Visual Studio": "16",
             "gcc": "7.3",
             "clang": "6",
-            "apple-clang": "10.0"
+            "apple-clang": "10.0",
         }
 
-        if compiler not in minimal_version:
-            self.output.info("%s requires a compiler that supports at least"
-                             " C++17" % self.name)
-            return
+    def package_id(self):
+        self.info.header_only()
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._min_cppstd)
+
+        if tools.Version(self.version) < "1.3.0":
```

Comment:
This means only versions before 1.3.0 need c++17.
Is this correct or should this become version after 1.3.0?

Reply:
correct, tabulate requires C++11 actually